### DISCUSSION TITLE
Install photoprism ubuntu with MariaDB

### DISCUSSION THREAD

**Discussion by johannessegl** [photoprism/photoprism#3328] [Q&A] [answered]
Hello, i found out how to install photoprism. my computer does not have so much storage so i connected a external usb HDD where i want to save all the photos. How can i install MariaDB and photoprism so that the photos (600000 photos), when i import them get saved on the HDD and not on the SSD where the system is?

I have a Pc with Ubuntu 22.04 and i run photoprism voa docker-compose (so docker)

Thanks in advance! I am despairing ;(

**Comment by eljejer** [comment]
I think, when you take a look at the yml file, and assign the originals folder a path to your mounted external disk, your wish will be granted

**Reply by johannessegl** [reply]
hey, thanks but i already tried that but i can not see the pictures on my mounted device

**Reply by graciousgrey** [reply]
How do your volume mounts look like? Do you see anything in the logs  when you try to index the files? 

Also make sure that you have the right file system permissions:

   https://docs.photoprism.app/getting-started/troubleshooting/docker/#file-permissions

**Reply by johannessegl** [reply]
i already fixed that problem. But do you know where i can find my uploaded fotos in the photoprism folder, because i need to backup them maybe in a cloud or sth

**Reply by graciousgrey** [reply, marked as answer]
If you use the [web upload](https://docs.photoprism.app/user-guide/library/upload/) or the [import functionality](https://docs.photoprism.app/user-guide/library/import/) photos will be uploaded to your configured originals directory in a year/month folder structure. Our backup guide can be found [here](https://docs.photoprism.app/user-guide/backups/).
